cmd/check.go

```diff
@@ -53,14 +55,23 @@ func (i *CheckCmd) Run(cmd *cobra.Command, args []string) {
 		panic(err)
 	}
 
+	fmt.Println("versions: ", versions)
```

Comment:
we can't print anything, if you need to, use the Log function to make sure it goes to stderr